src/Common/src/Interop/Windows/NtDll/Interop.NtQueryInformationProcess.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class NtDll
+    {
+        [DllImport(Libraries.NtDll, CharSet = CharSet.Unicode)]
+        internal static extern int NtQueryInformationProcess(SafeProcessHandle hProcess, PROCESSINFOCLASS pic, ref PROCESS_BASIC_INFORMATION pbi, int cb, ref int pSize);
+		
+		internal enum PROCESSINFOCLASS : int
+        {
+            ProcessBasicInformation = 0,
```

Comment:
This is a big enum we don't need most of, maybe just have what is documented for NtQueryInformationProcess and we can add more later if we need it?
ie
```
ProcessBasicInformation = 0
ProcessDebugPort = 7
ProcessWow64Information = 26
ProcessImageFileName = 27
ProcessBreakOnTermination = 29
ProcessSubsystemInformation = 75
```

Reply:
Sounds good to me! Change made!